examples/confluent_cloud_example/confluent_cloud_example.go

```diff
@@ -0,0 +1,106 @@
+// This is a simple example demonstrating how to produce a message to 
+// Confluent Cloud then read it back again.
+//     
+// https://www.confluent.io/confluent-cloud/
+// 
+// Confluent Cloud does not auto-create topics. You will need to use the ccloud 
+// cli to create the go-test-topic topic before running this example. The 
+// <ccloud bootstrap servers>, <ccloud key> and <ccloud secret> parameters are
+// available via the confluent cloud web interface. For more information,
+// refer to the quick-start:
+//     
+// https://docs.confluent.io/current/cloud-quickstart.html
+package main
+
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import (
+	"time"
+	"fmt"
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+	"os"
+)
+
+func main() {
+
+	p, err := kafka.NewProducer(&kafka.ConfigMap{
```

Comment:
same comments as for the Python client, so not repeating here.

Reply:
i was purposefully leaving the `api.version.request` in for safety since i was thinking librdkafka version was not guaranteed and as far as I could tell isn't enforced. but i'm also just as happy to take it out.

Reply:
It's been the default since v0.11.0, so I think we're fine without specifying it explicitly. We'll want people to use latest librdkafka anyway.